esp-box/esp-box.c

```diff
@@ -218,8 +219,8 @@ static lv_disp_t *bsp_display_lcd_init(void)
     const lvgl_port_display_cfg_t disp_cfg = {
         .io_handle = io_handle,
         .panel_handle = panel_handle,
-        .buffer_size = BSP_LCD_H_RES * 50,
-        .double_buffer = true,
+        .buffer_size = BSP_LCD_H_RES * BSP_LCD_DRAW_BUF_HEIGHT,
+        .double_buffer = BSP_LCD_DRAW_BUF_DOUBLE,
         .hres = BSP_LCD_H_RES,
         .vres = BSP_LCD_V_RES,
         .monochrome = false,
```

Comment:
I am sorry, I wasn't enough test rotation before, there are bad values in screen rotation initialization. There should be this:
```
/* Rotation values must be same as used in esp_lcd for initial settings of the screen */
        .rotation = {
            .swap_xy = false,
            .mirror_x = true,
            .mirror_y = true,
        },
```

Could you please fix in this review?

Reply:
ok, I will modify it tomorrow and submit it together

> I am sorry, I wasn't enough test rotation before, there are bad values in screen rotation initialization. There should be this:
> 
> ```
> /* Rotation values must be same as used in esp_lcd for initial settings of the screen */
>         .rotation = {
>             .swap_xy = false,
>             .mirror_x = true,
>             .mirror_y = true,
>         },
> ```
> 
> Could you please fix in this PR?

ok, I will modify it tomorrow and submit it together